addons/sys_rack/vrc103/__PREP__.sqf

```diff
@@ -0,0 +1,7 @@
+
+#include "script_component.hpp"
+TRACE_1("enter", _this);
+
+PREP_MODULE(vrc103,mountRadio103);
```

Comment:
Hmm, not 100% happy with the naming.

IMO it should be RackID first, function second. Like `acre_sys_rack_fnc_vrc103MountRadio`